client/src/components/App/Content/InvestigationForm/TabManagement/PersonalInfoTab/usePersonalInfoTab/index.ts

```diff
@@ -9,33 +11,69 @@ import Occupations from 'models/enums/Occupations';
 import { InvestigationStatus } from 'models/InvestigationStatus';
+import SubOccupationAndStreet from 'models/SubOccupationAndStreet';
+import investigatedPatientRole from 'models/investigatedPatientRole';
 import { setIsLoading } from 'redux/IsLoading/isLoadingActionCreators';
 import { PersonalInfoDbData } from 'models/Contexts/PersonalInfoContextData';
 import { setOccupations } from 'redux/Occupations/occupationsActionCreators';
 import InvestigationMainStatusCodes from 'models/enums/InvestigationMainStatusCodes';
-import { setInvestigatedPatientId } from 'redux/Investigation/investigationActionCreators';
 import useComplexitySwal from 'commons/InvestigationComplexity/ComplexityUtils/ComplexitySwal';
 
-import { usePersonalInfoTabParameters, usePersonalInfoTabOutcome } from './PersonalInfoTabInterfaces';
-import personalInfoValidationSchema from './PersonalInfoValidationSchema';
+import { PersonalInfoTabState } from '../PersonalInfoTabInterfaces';
+import personalInfoTabValidationSchema from '../PersonalInfoTabValidationSchema';
+import { usePersonalInfoTabIncome, usePersonalInfoTabOutcome } from './usePersonalInfoTabInterfaces';
 
-const usePersonalInfoTab = (parameters: usePersonalInfoTabParameters): usePersonalInfoTabOutcome => {
+const usePersonalInfoTab = (parameters: usePersonalInfoTabIncome): usePersonalInfoTabOutcome => {
+    
+    const [insuranceCompanies, setInsuranceCompanies] = useState<string[]>([]);
+    const [subOccupations, setSubOccupations] = useState<SubOccupationAndStreet[]>([]);
+    const [investigatedPatientRoles, setInvestigatedPatientRoles] = useState<investigatedPatientRole[]>([]);
 
     const { alertError } = useCustomSwal();
+    const { complexityErrorAlert } = useComplexitySwal();
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
-    const investigatedPatientId = useSelector<StoreStateType, number>(state => state.investigation.investigatedPatient.investigatedPatientId);
     const investigationStatus = useSelector<StoreStateType, InvestigationStatus>((state) => state.investigation.investigationStatus);
+    const investigatedPatientId = useSelector<StoreStateType, number>(state => state.investigation.investigatedPatient.investigatedPatientId);
 
-    const { complexityErrorAlert } = useComplexitySwal();
+    const getSubOccupations = (parentOccupation: string) => {
+        const subOccupationsLogger = logger.setup('Fetching Sub Occupation by Parent Occupation');
+        subOccupationsLogger.info(`launching sub occupations request with parameter: ${parentOccupation}`, Severity.LOW);
+        axios.get('/personalDetails/subOccupations?parentOccupation=' + parentOccupation).then((res: any) => {
+            if (res && res.data && res.data.data) {
+                subOccupationsLogger.info('got result from the DB', Severity.LOW);
+                setSubOccupations(res.data.data.allSubOccupations.nodes.map((node: any) => {
```

Comment:
move the data.data.allSubOccupations to the return in the server side or use a local constant like {nodes} = res.data.data.allSubOccupations